src/Blocks/components/form/assets/form.js

```diff
@@ -275,9 +359,17 @@ export class Form {
 		return Object.assign({}, { event: eventName, ...data });
 	}
 
+	// Scroll to specific element.
 	scrollToElement = (element) => {
 		if (element !== null) {
-			element.scrollIntoView(true);
+			element.scrollIntoView({block: 'start', behavior: 'smooth'});
 		}
 	}
+
+	// Dispatch custom event.
+	addEvent(element, name) {
```

Comment:
it's a bit weird naming.

Would `dispatchFormEvent` be more appropriate?

Reply:
I agree, dispatch sounds better, given that it dispatches the event.